analyses/org.eclipse.tracecompass.incubator.inandout.core/schema/in-and-out-analysis.json

```diff
@@ -0,0 +1,59 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://org.eclipse.tracecompass/in-and-out-analysis.json",
+    "title": "InAndOut Analysis",
+    "description": "Custom Execution Analysis schema",
+    "type": "object",
+    "properties": {
+        "name" : {
+            "type": "string",
+            "description": "The name of this InAndOut configuration"
+        },
+        "description" : {
+            "type": "string",
+            "description": "The descrition of this InAndOut configuration"
+        },
+        "specifiers": {
+            "description": "array specifiers",
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "label": {
+                        "type": "string",
+                        "descritpion": "The label of the identifier"
```

Comment:
A useful optional field here (apart from type and description) would be a list of values to select elements from. This could be used to provide an option to the user to select instead of type in.

Reply:
Yes, we have to follow the JSON schema specification to see what is possible (https://json-schema.org/). The designer of the schema and extension will have look into that. I know, that you, for example, can specify a regex pattern to limit the input.